changelogs/unreleased/55668.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "FEC Import : fix invoice term generation"
```

Comment:
Fix invoice term ? Invoice term does not exist on dev branch so I don't get the purpose of this change on 6.3 branch